### DISCUSSION TITLE
admin app resources request takes too long ~1min

### DISCUSSION THREAD

**Discussion by utopictown** [keycloak/keycloak#33365] [Q&A]
some requests took too long to finish. fresh keycloak install, 1 realm other than master, no users at all.

hosted on 2vcpu, 4GB with docker stack
`image: quay.io/keycloak/keycloak:25.0.6`

can anyone help figure why this happen?

![keycloak 1 min](https://github.com/user-attachments/assets/1a0e35c7-0b19-4489-9716-e0d4f19b70ba)

**Comment by codespearhead** [comment]
Show us the configuration you passed to the Keycloak container.
